### ISSUE TITLE
Data, data, data

### ISSUE THREAD

**Issue by reginafcompton** [datamade/csh-platform#3]
I put together tree diagrams. Doing so raised a couple thoughts/questions.

* _When should we ask about location?_ This should be the very first question, or more specifically, a location should be the "trigger" word that launches the questionnaire. We want users to answer the least number of questions, and it seems impossible to recommend a resource without knowing using location. Plus, the suburbs have only two options, so users in the suburbs need only answer a couple questions, before reaching a state of termination. (We'd need to know they were in the suburbs to build out the shorter paths.)

* I see three question-and-answer paths that lead to two sources:

    **18-24; slept at my home or that of a friend/family; unstable housing situation** 
Coordinate Entry Access points
Youth Diversion Supports (the logic table marks this one as being prioritized before CEA points)

    **18-24; slept outside** 
Coordinate Entry Access
Chicago Shelter System

    **25+; slept outside**
Coordinate Entry Access
Chicago Shelter System

Or should those last two paths _only_ recommend CES points? And the Chicago Shelter System should be recommend to users who pass through all states without receiving a specified source, right?

**Comment by reginafcompton** [comment]
[city_paths.pdf](https://github.com/datamade/csh-platform/files/2263148/city_paths.pdf)
[suburban_paths.pdf](https://github.com/datamade/csh-platform/files/2263149/suburban_paths.pdf)

**Comment by reginafcompton** [comment]
### Logic table data

['The Bad Idea for “Fixing” Homelessness That Just. Won’t. Die.'](https://medium.com/@candacefaber/the-bad-idea-for-fixing-homelessness-that-just-wont-die-7f26a1f3ac4f) (Article shared by @derekeder!)

This article (however emotionally charged) raises a good question about the CSH data - we assume that users are individuals. Is this always the case? Or should we ask a question about children? spouses? pets? Would the recommended resource change, if say, an user had a kid or a dog?

**Comment by fgregg** [comment]
Great question! Let's be sure to raise this with the client.

**Comment by reginafcompton** [comment]
No, the survey outcome does not change if the user has a kid, spouse, or dog.
